src/alfasim_sdk/_internal/alfacase/alfatable.py

```diff
@@ -33,11 +33,12 @@ def load_pvt_model_table_parameters_description_from_alfatable(
     """
     Load the content from the alfatable in the given file_path. The validation is turned off due to performance issues.
     """
-    from ruamel import yaml as ruamelyaml
+    from ruamel.yaml.main import YAML
     from barril.units import Scalar
     import numpy as np
 
-    content = ruamelyaml.safe_load(Path(file_path).read_text(encoding="UTF-8"))
```

Comment:
Don't you need to set an encoding?
Did you try to open a alfacase with utf8 encoding?

Reply:
Well it seems that we're lacking tests for "open a alfacase with utf8 encoding", because all tests passed with this change.

Anyway since this is minor I'll tackle it in another moment.